config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show]
+  
+  resources :chefs, only: [:show] do
+    get 'ingredients', to: 'ingredients#index'
```

Comment:
ThYou mentioned feeling like your route was not quite right, and you might have been thinking about the possibility of a nested controller. Using the ingredients controller like you have here is reasonable, but another alternative could be to create a `chefs` directory in your controllers folder and adding the ingredients controller there. Then, this route would go to `"chefs/ingredients#index"` so you would need to add the `controller: "chefs/ingredients"` parameter to this line on your route. Either this or your implementation are okay for this IC. But on a project, if you needed to have an ingredients controller that showed ALL ingredients, you would want to use "ingredients#index" for that so this US for a chef's ingredients could go to `"chefs/ingredients#index"`

Let me know if any of this is unclear!